docs/index.md

```diff
@@ -1378,11 +1378,11 @@ tests as shown below:
     <title>Mocha</title>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://unpkg.com/mocha@5.3.0/mocha.css" />
+    <link rel="stylesheet" href="https://unpkg.com/mocha@6.0.0/mocha.css" />
   </head>
   <body>
     <div id="mocha"></div>
-    <script src="https://unpkg.com/mocha@5.3.0/mocha.js"></script>
+    <script src="https://unpkg.com/mocha@6.0.0/mocha.js"></script>
```

Comment:
we can actually use markdown-magic here to insert the current version of mocha by pulling `version` out of `package.json`

Reply:
Opened #3603 for this. Mocha wasn't using `markdown-magic` until 4 days ago.